### DISCUSSION TITLE
google and cloudflare are trying to kill non-mainstream browsers.

### DISCUSSION THREAD

**Discussion by crocket** [qutebrowser/qutebrowser#6698] [General]
Google prevented qutebrowser from logging in. Cloudflare used to block non-mainstream browsers and may still do.

This is why I try to stay away from google and cloudflare as much as possible.

**Comment by pipelinedave** [comment]
I have not heard of this.
Can you share any source for these claims? Mind you I am not in disbelief of your claims, its just that I have not experienced any issues with either Google or Cloudflare using qutebrowser, personally.

**Reply by toofar** [reply]
See here for relevant adaptations we have had to make I suppose https://github.com/qutebrowser/qutebrowser/issues/5182
